src/models/event.js

```diff
@@ -2,7 +2,7 @@ const mongoose = require('mongoose');
 const variables = require('../utility/variables');
 
 const eventSchema = mongoose.Schema({
-    puplisherId: {
+    publisherId: {
```

Comment:
it took me one hour of my life to discover this typo! 
I CRY

Reply:
Kanka,  I CRY too 😭